caikit/runtime/interceptors/caikit_runtime_server_wrapper.py

```diff
@@ -137,17 +138,18 @@ def safe_rpc_wrapper(rpc):
             A function that takes a gRPC request message and ServicerContext, and safely invokes
             the provided RPC.
         """
-
-        if rpc is not None:
-            if rpc.__name__ == "safe_rpc_call":
-                return rpc
-            log.info(
-                "<RUN33333123I>",
-                "Wrapping safe rpc for %s",
-                rpc.__name__,
-            )
-        else:
-            log.info("<RUN33322123I>", "Wrapping safe rpc for %s", rpc)
+        if rpc is None:
```

Comment:
Is this the "nasty bug fix" you were talking about? Trying to parse what actually changed here